benchmarks/lib/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 val accpVersion: String? by project
 val accpLocalJar: String by project
 val fips: Boolean by project
+val includeBenchmark: String by project
```

Comment:
Would be more general if you could specify multiple benchmarks.

Reply:
This string is can be a regular expression. I updated the README to reflect this fact.